src/features/virtualized-table-renderers/__tests__/draggableRowRenderer.test.js

```diff
@@ -31,7 +31,7 @@ describe('features/virtualized-table-renderers/draggableRowRenderer', () => {
         };
         draggableProvided = { draggableProps: {}, dragHandleProps: {} };
         draggableSnapshot = { isDragging: false };
-        wrapper = getWrapper(rowRendererParams);
+        wrapper = getWrapper(rowRendererParams).dive();
```

Comment:
DragDropContext is now rendering ErrorBoundary as a top most component so I'm using dive to skip one level. This way I don't need to modify the rest of the code.

Reply:
would it work to add `.dive()` to the `shallow` in the implementation of `getWrapper` so that it doesn't need to be added to the individual instances?